swagger.yaml

```diff
@@ -580,9 +580,7 @@ definitions:
             properties:
               MsgId:
                 type: string
-                pattern: '^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$'
-                minLength: 36
-                maxLength: 36
+                maxLength: 35
```

Comment:
For the min length stuff the reason why i changed it was because the test data has less than 35 characters . I will change the test data to reflect 35 characters in length and add back the minlength validation